lesson1/word_reverse.js

```diff
@@ -0,0 +1,11 @@
+function reverseWord(word) {
+    let reversed = ''
+
+    for(let i = word.length - 1; i >= 0; i--) {
```

Comment:
Не обязательно так усложнять цикл =) Достаточно инициализироваться с числа, которое будет конечным:
```
let i = word.length;
```
И условием выхода обозначить достижение нуля:
```
i > 0
```
Итого:
```suggestion
    for(let i = word.length; i > 0; i--) {
```